scripts/artis-kelvin2.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash -x
+#SBATCH --ntasks=128
+#SBATCH --ntasks-per-node=128
+#SBATCH --time=00:30:00
```

Comment:
We usually aim for about ~1000 cores and 24 hours by default with these job scripts. Is this possible on Kelvin?